docsrc/conf.py

```diff
@@ -535,6 +535,7 @@
         'cyrus-3.4':('https://www.cyrusimap.org/3.4%s',None),
         'cyrus-3.6':('https://www.cyrusimap.org/3.6%s',None),
         'cyrus-3.8':('https://www.cyrusimap.org/3.8%s',None),
+        'cyrus-3.9':('https://www.cyrusimap.org/3.9%s',None),
```

Comment:
This section is only for stable releases, which since 3.0 are only even-numbered.  3.9 does not belong here; there is no `/3.9` tree on the website.

The documentation for the current development version is at `/dev`, which is taken care of by `'cyrus-dev'` a line below.